lib/utils/log/levels.go

```diff
@@ -42,3 +53,26 @@ func SlogLevelToLogrusLevel(level slog.Level) logrus.Level {
 		return logrus.FatalLevel
 	}
 }
+
+// UnmarshalText unmarshals log level text representation to slog.Level.
+func UnmarshalText(data []byte) (slog.Level, error) {
```

Comment:
I'm not sure if I understood what you meant. But the idea here is to avoid having an addition type (say `Level`) that we could implement in the encode/decode interfaces, given that we're already relying on the `slog` package. However, I don't have a strong opinion on doing this. If you have any suggestions, let me know.

Reply:
I see. I think moving it to the debug package makes more sense.